src/styles/styles.js

```diff
@@ -1100,6 +1101,19 @@ const styles = {
         lineHeight: variables.fontSizeOnlyEmojisHeight,
     },
 
+    emojiMessageText: {
+        position: 'relative',
+        fontSize: variables.fontSizeEmoji,
+        lineHeight: variables.fontSizeEmojiHeight,
+        top: Platform.OS === 'android' ? 8 : 3,
```

Comment:
I'm not sure if styles are different per platform, they should be splinted into separate folders like other styles, for android index.android.js, index.ios.js for ios and index.js for the web.
@mountiny, could you correct me if I'm wrong?

Reply:
Hi @parasharrajat relative to this comment
I was asked to separate the conditional styles into index.(ios | android | native).js which I am proposing is not so necessary for a small set of styles that is only applicable to emoji rendering, my reasons are creating files for both android, ios and web just to render emoji also increases  app size and for such a simple conditional style we can make way with Platform.select which is far better for this use case
I made a recent change on my last push to the open PR, which conditionally renders emoji. This is better than creating files just for a simple set of styles, what do you think?
 
 ```javascript
   emojiMessageText: {
        position: 'relative',
        fontSize: variables.fontSizeEmoji,
        lineHeight: variables.fontSizeEmojiHeight,
        ...Platform.select({
            ios: {
                top: 8,
            },
            android: {
                top: 8,
            },
            default: {
                top: 3,
            },
        }),
    },
 ```